deployer.go

```diff
@@ -121,9 +121,7 @@ func (d *deployer) deployUnit(wantedUnit *schema.Unit) error {
 
 	wuf := schema.MapSchemaUnitOptionsToUnitFile(wantedUnit.Options)
 	cuf := schema.MapSchemaUnitOptionsToUnitFile(currentUnit.Options)
-	wufUnescapedString := strings.Replace(wuf.String(), "\\\n", "", -1)
-	if strings.Replace(wufUnescapedString, " ", "", -1) != strings.Replace(cuf.String(), " ", "", -1) {
-
+	if wuf.Hash() != cuf.Hash() {
```

Comment:
Well thats annoying! Please leave this unmerged and I'll do some testing next week, want to understand it

On 19 June 2015 14:13:03 BST, Catalin Ilea <notifications@github.com> wrote:
>> @@ -121,9 +121,7 @@ func (d *deployer) deployUnit(wantedUnit
>*schema.Unit) error {
>>  
>>  	wuf := schema.MapSchemaUnitOptionsToUnitFile(wantedUnit.Options)
>>  	cuf := schema.MapSchemaUnitOptionsToUnitFile(currentUnit.Options)
>> -	wufUnescapedString := strings.Replace(wuf.String(), "\\\n", "", -1)
>> -	if strings.Replace(wufUnescapedString, " ", "", -1) !=
>strings.Replace(cuf.String(), " ", "", -1) {
>> -
>> +	if wuf.Hash() != cuf.Hash() {
>
>It will do that every time actually, not just once.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/Financial-Times/coco-fleet-deployer/pull/4/files#r32826896

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.